instructions.md

```diff
@@ -0,0 +1,58 @@
+# #TODO 1
```

Comment:
great idea!

you can name this file `lessons-learned`.

Reply:
accepted